### PR TITLE
Created just in time component

### PR DESCRIPTION
**Pull request by deniselashlley**
feat: Added the Just In Time Message component

### CODE REVIEW

File: src/components/JustInTime/JustInTime.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+    );
+  }
+}
+
+JustInTime.propTypes = {
+  linkText: propTypes.string.isRequired,
+};
+
+export default JustInTime;
```
Comment (Heleen-cr) [inline src/components/JustInTime/JustInTime.js]: should we give this an id property to use with the href so it's possible to use multiple JIT messages on a page?

File: src/components/JustInTime/JustInTime.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+/* Pattern Lab Comic Relief theme variables */
+@import "~@comicrelief/pattern-lab/sass/themes/cr/2017/variables/mixins";
+
+@import "~@comicrelief/pattern-lab/sass/base/components/just-in-time-block/_just-in-time";
+
+.form__row--just-in-time-block {
+  .close-link {
+    display: inline
+  }
+}
```
Comment (Heleen-cr) [inline src/components/JustInTime/JustInTime.scss]: new line

**Comment by Heleen-cr** [comment]
I don't think we're committing yarn lock files for components in storybook by the way. Should we?

File: src/components/JustInTime/JustInTime.js (hunk 1 above)
Comment (deniselashlley) [reply src/components/JustInTime/JustInTime.js]: @Heleen-cr I asked this question and it was not likely to happen

Comment (Heleen-cr) [reply src/components/JustInTime/JustInTime.js]: ah ok that's cool then

**Pull request merged**